starknet_py/tests/e2e/account/account_test.py

```diff
@@ -13,6 +13,7 @@
     Call,
     DeployAccountTransaction,
```

Comment:
Should we conform already existing names to new ones?

Reply:
Work in progress in #1259